src/styles/themes.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { DefaultTheme } from '@react-navigation/native';
+import { Platform } from 'react-native';
+
+import { _PRIMARY_COLOR as PRIMARY_COLOR } from '../config';
+
+/**
+ * Light theme color scheme
+ * @type {{
+ * midContrastDetail: string,
+ * errorBgColor: string,
+ * backgroundColor: string,
+ * borderColor: string,
+ * lowContrastDetail: string,
+ * textColorShadow: string,
+ * positiveBalanceColor: string,
+ * textColor: string,
+ * errorTextColor: string,
+ * tabBarBackground: string,
+ * primaryOpacity30: string,
+ * textColorShadowLight: string,
+ * primaryOpacity10: string,
+ * errorTextShadow: string,
+ * primary: string}}
+ * @property {string} backgroundColor The main background color
+ * @property {string} lowContrastDetail Low contrast with background: separator lines, containers...
+ * @property {string} midContrastDetail Medium contrast with background: placeholders, ...
+ * @property {string} borderColor Defines borders
+ * @property {string} textColor Maximum contrast with the background color, for reading
+ * @property {string} textColorShadow Washed down version of the text
+ * @property {string} textColorShadowLight More washed down version of the text
+ * @property {string} tabBarBackground Specific for tab bar selectors
+ * @property {string} positiveBalanceColor Represents a positive feedback for the user
+ * @property {string} errorBgColor For containers with error feedbacks
+ * @property {string} errorTextColor For texts with error messages
+ * @property {string} errorTextShadow Washed down version of error texts
+ * @property {string} primary Primary color, as set on the application config file
+ * @property {string} primaryOpacity10 Primary color washed down to 10% opacity
+ * @property {string} primaryOpacity30 Primary color washed down to 30% opacity
+ */
+export const COLORS = {
+  backgroundColor: '#fff',
```

Comment:
The current code has a different background color for iOS and Android. Why do you have only one now?

Reply:
Previously there was an effort to match the background color with the header color and the status bar on top. For reference, the android color was `#fafafa`, hardly noticeable from `#ffffff`.

Since the removal of the background color on the Header ( see "Dashboard" and "Settings" screenshots on the PR description, and the `HathorHeader.js` line 84 removing the background color ), there is no need to try and match with such precision the background color with the OS because of this outer border that contains the "lowContrast" color.

Having the same background color also makes the app presentation more consistent between operating systems.

What do you think about this?

Reply:
I just need to see the difference between the screens now and before. Do you have a screenshot comparing old and new screen for both iOS and Android?